### PR TITLE
Add Ethereum Inscription Mainnet

### PR DESCRIPTION
**Pull request by ETINSDev**
This PR adds the chain file for the Ethereum Inscription Mainnet.

To learn more about Ethereum Inscription, you can check out:
Website: https://www.etins.org/

### CODE REVIEW

**Comment by ETINSDev** [comment]
@ligi Hello friends, can you help me deal with it? ""This branch is out-of-date with the base branch" appears again.

File: _data/chains/eip155-1617.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "features": [{ "name": "EIP155" }, { "name": "EIP1559" }],
+  "infoURL": "https://www.etins.org",
+  "shortName": "etins",
+  "chainId": 1617,
+  "networkId": 1617,
+  "explorers": [
+    {
+      "name": "Ethereum Inscription Explorer",
+      "url": "https://explorer.etins.org",
+      "standard": "EIP3091"
```
Comment (ligi) [inline _data/chains/eip155-1617.json:20]:
```suggestion
      "standard": "none"
```

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-1617.json (hunk 1 above)
Comment (ligi) [reply _data/chains/eip155-1617.json:20]: your explorer is not EIP3091 compatible

**Comment by ETINSDev** [comment]
Thank you very much

**Pull request merged**